logger_test.go

```diff
@@ -95,3 +101,123 @@ func TestLogger_SetBufferPool(t *testing.T) {
 	assert.Equal(t, pool.get, 1, "Logger.SetBufferPool(): The BufferPool.Get() must be called")
 	assert.Len(t, pool.buffers, 1, "Logger.SetBufferPool(): The BufferPool.Put() must be called")
 }
+
+func TestLogger_concurrentLock(t *testing.T) {
+	SetFormatter(&LogFormatter{})
+	go func() {
+		for {
+			func() {
+				defer func() {
+					// 处理所有异常，防止panic导致程序关闭
```

Comment:
What does this comment says?

Reply:
sry I have deleted unnecessary Chinese comments